tools/win/finish-windows.ps1

```
@@ -0,0 +1,174 @@
+param([string]$OutDir = '')
+
+$ErrorActionPreference = 'Stop'
+$repo = Split-Path (Split-Path $PSScriptRoot -Parent) -Parent
+if ([string]::IsNullOrWhiteSpace($OutDir)) { $OutDir = Join-Path $repo 'captures' }
+New-Item -ItemType Directory -Force -Path $OutDir | Out-Null
+
+# Transcript rather than output redirection: this script talks to the operator
+# with Write-Host, which bypasses the pipeline, so redirecting stdout would
+# capture nothing AND hide the countdowns they need to follow.
+$transcript = Join-Path $OutDir 'finish-windows.log'
+try { Start-Transcript -Path $transcript -Force | Out-Null } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)